translations/ui-agreements/en_US.json

```diff
@@ -203,6 +203,7 @@
     "eresourcesCovered.exportAsKBART": "Export as KBART",
     "eresourcesCovered.json": "JSON",
     "eresourcesCovered.kbart": "KBART",
+    "eresourcesCovered.exportButton.tooltip": "Exports only supported for current or all resources",
```

Comment:
Don't make _additions_ to translation files other than `en`. It's not necessary because of the way the fallbacks work and it breaks the Localize workflow.

It is however fine to make _changes_ to existing keys in en_US so that the correct string shows up.